<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Сотрудники</string>
     </attribute>
     <widget class="QPushButton" name="addemployee">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Добавить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>plus.png</normaloff>plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="editemployee">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите элемент для редактирования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pngwing.com.png</normaloff>pngwing.com.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteemployee">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Удалить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>minus.png</normaloff>minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="picture">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>100</y>
        <width>350</width>
        <height>450</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Сортировать по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort1_emp">
           <item>
            <property name="text">
             <string>Id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Возрасту</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Алфавиту</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort2_emp">
           <item>
            <property name="text">
             <string>По возрастанию</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По убыванию</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Найти:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_line"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Группы</string>
     </attribute>
     <widget class="QTableWidget" name="tablegroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_43">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_44">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Сортировать по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort1_gr">
           <item>
            <property name="text">
             <string>Id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Алфавиту</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort2_gr">
           <item>
            <property name="text">
             <string>По возрастанию</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По убыванию</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_45">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>Найти:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_16"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addgroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Добавить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>plus.png</normaloff>plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="editgroup">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите элемент для редактирования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pngwing.com.png</normaloff>pngwing.com.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="deletegroup">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Удалить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>minus.png</normaloff>minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Отделы</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_52">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_53">
         <item>
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>Сортировать по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort1_dep">
           <item>
            <property name="text">
             <string>Id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Алфавиту</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort2_dep">
           <item>
            <property name="text">
             <string>По возрастанию</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По убыванию</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_54">
         <item>
          <widget class="QLabel" name="label_38">
           <property name="text">
            <string>Найти:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_19"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tabledep">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="deletedep">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Удалить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>minus.png</normaloff>minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="adddep">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Добавить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>plus.png</normaloff>plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="editdep">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите элемент для редактирования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pngwing.com.png</normaloff>pngwing.com.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Должности</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_55">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_56">
         <item>
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>Сортировать по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort1_pos">
           <item>
            <property name="text">
             <string>Id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Алфавиту</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sort2_pos">
           <item>
            <property name="text">
             <string>По возрастанию</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По убыванию</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_57">
         <item>
          <widget class="QLabel" name="label_40">
           <property name="text">
            <string>Найти:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_20"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tablepos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="deletepos">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Удалить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>minus.png</normaloff>minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="addpos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Добавить&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>plus.png</normaloff>plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="editpos">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите элемент для редактирования&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pngwing.com.png</normaloff>pngwing.com.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_5">
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Избранное</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Обычная</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Темная</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
